package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "edx-ora2",
-  "version": "4.4.3",
```

Comment:
same as https://github.com/openedx/edx-bulk-grades/pull/125#discussion_r934834568 (although I suppose you might want to synch the version numbers anyway here)